pkg/nodeconfig/payload/payload.go

```diff
@@ -63,6 +65,89 @@ const (
 	// AzureCloudNodeManagerPath contains the path of the azure cloud node manager binary. The container image should
 	// already have this binary mounted
 	AzureCloudNodeManagerPath = payloadDirectory + AzureCloudNodeManager
+	// networkConfTemplate is the template used to generate the network configuration script
+	networkConfTemplate = `# This script ensures the contents of the CNI config file is correct, and returns the HNS endpoint IP
```

Comment:
It is fine to leave this as a file. I had only asked for the in-lining in the test case. Sorry if that was not clear.

Reply:
That's fine with me as long as it is easy to maintain.